### ISSUE TITLE
Start not working

### ISSUE THREAD

**Issue by Heryurec** [qconrad/intellistar-emulator#58]
Whenever I press start, nothing happens. What do I do?

**Comment by tjhorner** [comment]
It seems that weather.com has started enforcing stricter CORS settings, so the browser refuses to load data from it.

**Comment by fdezwonders** [comment]
@tjhorner No, it's just that the API key that qconrad uses for their emulator is not working.
@Heryurec What you can do is to try using [a different fork](https://dalk21.github.io/intellistar-emulator/) or forking the repo and changing the API key in "config.js".

**Comment by Heryurec** [comment]
How do I get an API if I don't have a weather station at my home?

**Comment by Heryurec** [comment]
How do I also get the music working

**Comment by fdezwonders** [comment]
@Heryurec As I explained: try changing The Weather Channel API key, or, if you don't have one, try using a different fork, such as [this one](https://dalk21.github.io/intellistar-emulator/).

(P.S: If you need to ask another question after you already commented your first question, just edit the first question's comment.
And please check other issues before creating a new one, someone already asked this in #53.)
